docs/about/development/testing.md

```diff
@@ -13,9 +13,12 @@ There are a few handy targets in the Makefile to help you run tests:
   - Note: this runs with a full clean/rebuild, so it is not great for fast iteration.
     Use `make e2e_test` to pick individual tests from the Cypress UI.
 - `make test`: Run e2e, client- and server-side testing suites.
+- `yarn playwright test`: Runs the playwright tests against the dec server and useful for inspecting individual e2e tests when debugging.
```

Comment:
**question:** Do you mean dev server? 
**suggestion:** I also split the second part of the sentence since it's a separate idea from what the command does.

```suggestion
- `yarn playwright test`: Runs the playwright tests against the dev server. Once started, one can inspect the individual e2e tests when debugging.
```